### DISCUSSION TITLE
Microsoft can't open the ganache app

### DISCUSSION THREAD

**Discussion by Rodrigopaiva44** [ConsenSys-archive/truffle#5955] [Installation] [answered]
Hi guys, i'm trying to open the app in my windows but the following message appears:
![2023-03-07](https://user-images.githubusercontent.com/73554373/223562132-c3e2e13a-a18e-4bbf-9eb7-efa1e817cda0.png)

**Comment by Rodrigopaiva44** [comment, marked as answer]
Even if i try to open as an admin or reinstalling the program...

**Reply by FlaviuWeb3** [reply]
Hi @Rodrigopaiva44 could you please install our latest version from this link -> https://github.com/trufflesuite/ganache-ui/releases/download/v2.7.0/Ganache-2.7.0-win-x64-setup.exe? Let me know how it works.

**Reply by Rodrigopaiva44** [reply]
Hi Flaviu, actually i did installed the latest version (2.7)
